src/Microsoft.AspNet.Mvc.Core/Controllers/FilterActionInvoker.cs

```diff
@@ -587,7 +587,7 @@ private Task InvokeAllExceptionFiltersAsync()
                         {
                             _telemetry.WriteTelemetry(
                                 "Microsoft.AspNet.Mvc.AfterActionMethod",
-                                new { actionContext = ActionContext, result });
+                                new { actionContext = ActionContext, result = result });
```

Comment:
using `new { result }` is legal c#, but not a practice we should use with event notifications. The parameter names are significant, and we don't want to break consumers with a refactor